scss/_utilities.scss

```diff
@@ -443,16 +443,17 @@ $utilities: map-merge(
       class: font,
       values: (monospace: $font-family-monospace)
     ),
-    "rounded": (
+    "border-radius": (
```

Comment:
Thoughts from @twbs/css-review for using `radius`, `br`, or something else shorter here? If we go with `br`, I’d suggest we change `border` to  ` b` as well. Good example of this is `bg` for `background-color`.

Reply:
Yup, `br` and `b` make sense.